roles/kubernetes/master/templates/kubeadm-config.v1beta1.yaml.j2

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubeadm.k8s.io/v1beta1
+apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
How about the file name?

Reply:
Indeed, do you want me to update it to v1beta2.yaml ?

Reply:
There already is a v1beta2, or am I missing something?

Reply:
@holmsten oh man, not you it's me that miss something!